content/web-frameworks/gatsby/why-gatsby.md

```diff
@@ -31,6 +35,17 @@ showToc: true
   - [GitHub Action workflow for GitHub Pages](/programming-cookbook/github-actions/deploy-to-github-pages)
   - [GitHub Action workflow for CloudFlare Workers](/programming-cookbook/github-actions/deploy-to-cloudflare-workers)
 
+# Gatsby don't be suitable if:
+
+- The web admin is not a developer. To modify the content or design, it requires programming skill.
+- You want admin control panel to modify the content.
+- Your content will be changed quite often, it'll make difficult to modify the content when compare to use admin panel.
+  - If you want to update the content in mobile or public computer, it might not convenience. The computer requires specify tools such as an editor, git command etc.
+- You're sticky on the traditional technology like basic html and css renderer tools. Other tools will be suitable for you such as:
```

Comment:
Very minor stick is a verb, think we should say You stick to or change you You rely on.
Please also use CAPITAL for abbreviation, HTML, CSS.

Reply:
Thanks, I'll fix it.